src/LibraryManagement.Client/UI/src/app/authentication/login/login.component.ts

```diff
@@ -1,23 +1,27 @@
 import { Component } from '@angular/core';
-import { UserLogin } from '../models/user-login';
 import { HttpClient } from '@angular/common/http';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.sass']
 })
 export class LoginComponent {
-  errors = new Array();
-  user = new UserLogin();
+  loginForm = new FormGroup({
+    email: new FormControl("", [Validators.required, Validators.email]),
+    password: new FormControl("", Validators.required)
+  });
 
   constructor(private httpClient: HttpClient) {}
 
   protected onSubmit() {
-    this.httpClient.post("http://localhost:7141/api/authentication/login", this.user)
+    this.httpClient.post("https://localhost:7141/api/authentication/login", this.loginForm.value)
```

Comment:
You should set a variable to save base URL API in app settings

Reply:
Will do in next PR